### DISCUSSION TITLE
Temp setting

### DISCUSSION THREAD

**Discussion by n6812t** [mriscoc/Ender3V2S1#791] [Q&A] [answered]
My apologies if I had posted this elsewhere, I can't find it now.  Very new to this, and this is my first foray into github as well.  

I successfully installed the firmware on 3s1 with F4 chip.  The temp is obviously limited to 265.   I then found where the "fix" is to create a text file with the following in it:

C104 U1 T300

and name it with an extension of gco or gcode.  I tried both.  When I try to "print" this file to execute it, the file cannot be found.  Other gcode files are visible to the machine.  Is there a trick to creating the file that I'm missing?  I haven't had this issue with a CNC mill that I also use.  Befuddled.

Thank you in advance!

**Comment by mriscoc** [comment, marked as answer]
Some text editors adds a .txt hidden extension at the end of file name.

**Reply by n6812t** [reply]
That was probably it.  I ended up opening an old gcode file in notepad, emptying it, and adding the code.  Worked great.  Many thanks!!
